mapping_project/lib/saved_locs.dart

```diff
@@ -16,7 +17,11 @@ class SavedLocs extends StatefulWidget {
 class _SavedLocsState extends State<SavedLocs> {
   //Actual savedlocs ofc
 
-  var userLocs = ["Room 1", "Room 2", "Room 3"];
+  var userLocs = [
```

Comment:
Can you not make this original data come from a file - and if it's empty just return something like "you have not saved any rooms"?